files/image_config/caclmgrd/caclmgrd

```diff
@@ -259,6 +259,9 @@ class ControlPlaneAclManager(object):
         iptables_cmds.append("ip6tables -A INPUT -p tcp --dport 179 -j ACCEPT")
         iptables_cmds.append("ip6tables -A INPUT -p tcp --sport 179 -j ACCEPT")
 
+        # Add iptables/ip6tables commands to allow all restapi/http/https requests
+        iptables_cmds.append("iptables -A INPUT -p tcp --dport 8081 -j ACCEPT")
+        iptables_cmds.append("iptables -A INPUT -p tcp --dport 8090 -j ACCEPT")
```

Comment:
can you check if the FEATURE table has restapi service, if yes, then enable these ports?

Reply:
currently restapi is not in the FEATURE table. Once we add, may be we can revisit this later.

Reply:
It has been added: https://github.com/Azure/sonic-buildimage/blob/master/files/build_templates/init_cfg.json.j2#L33

Reply:
I see, but that change is not planned for 201911 right?

Reply:
It will be backported.

Reply:
i see, but still i dont think it needs to be installed based on feature status. Also, feature can be enabled disabled and not sure we want to add/delete iptable rules based on that

Reply:
Why not?

Reply:
can you tell the flow when feature is enabled/disabled dynamically? you mean, restart caclmgrd or do we have a way to insert rules?

Reply:
Currently, it is not being done. However, caclmgrd could also listen for changes to the FEATURE table as well as the ACL tables.